contentcuration/contentcuration/frontend/channelList/views/KCard/index.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+  >
+    <div>
+      <KImg
+        v-if="layout === 'vertical' && thumbnailDisplay !== 'none'"
+        :src="thumbnailSrc"
+        :height="300"
+        :width="500"
+        :isDecorative="false"
+        altText="thumbnail image"
+        :appearanceOverrides="{
+          objectFit: thumbnailScaleType,
+          backgroundColor: $themeTokens.fineLine
+        }"
+        :style="KImgVerticalSmallStyle"
+      />
+      <slot v-if="!thumbnailSrc" name="thumbnailPlaceholder"></slot>
+
+      <div class="horizontal-layout-style">
+
+        <KImg
+          v-if="layout === 'horizontal' && thumbnailDisplay === 'large'"
+          :src="thumbnailSrc"
+          :height="300"
+          :width="600"
+          :isDecorative="false"
+          altText="thumbnail image"
+          :appearanceOverrides="{
+            objectFit: thumbnailScaleType,
+            backgroundColor: $themeTokens.fineLine
+          }"
+        />
+        <slot v-if="!thumbnailSrc" name="thumbnailPlaceholder"></slot>
+
+        <div>
+          <KRouterLink
```

Comment:
[Required] Resulting card's html markup doesn't need to (and even shouldn't, because of a11y) have two (in this case very same) links. The link that wraps the title text in the `BaseCard` is the only one we need.

After any markup changes I think it could be helpful to inspect html in the browser console and see if it makes sense semantically. Then you can tweak styles and JavaScript to enhance behavior to fit the specification, while keeping the semantic html structure meaningful. This is useful frontend development pattern, commonly called accessibility-first development.